.travis.yml

```diff
@@ -34,6 +34,28 @@ linux-s390x: &linux-s390x
             - libpcre3-dev
             - mercurial
             - libpq-dev
+            
+              install:
```

Comment:
`install:` block should be consistent with the indentation of `addons:`

Reply:
Missed this, thanks for the suggestion.
The indentation is in line with `addons:`